lib/reports/content_change_email_status_count.rb

```diff
@@ -1,34 +1,32 @@
 module Reports
   class ContentChangeEmailStatusCount
-    def initialize(content_change)
-      @content_change = content_change
-      @email_status_count = email_status_count
+    def initialize(content_changes)
+      @content_changes = content_changes
     end
 
     def self.call(*args)
       new(*args).call
     end
 
     def call
-      puts <<~TEXT
-        -------------------------------------------
-        Email status counts for Content Change #{@content_change.id}
-        -------------------------------------------
-
-        Sent emails: #{@email_status_count['sent']}
-
-        Pending emails: #{@email_status_count['pending']}
-
-        Failed emails: #{@email_status_count['failed']}
-
-        -------------------------------------------
-      TEXT
+      @content_changes.each do |content_change|
+        puts <<~TEXT
+
+          ---------------------------------------------------------------------------
+          Email status counts for Content Change #{content_change.id}
+          ---------------------------------------------------------------------------
+          Sent emails: #{email_status_count(content_change)['sent']}
+          Pending emails: #{email_status_count(content_change)['pending']}
+          Failed emails: #{email_status_count(content_change)['failed']}
```

Comment:
We should save the results of this call because right now it means we'll be running the query three times for each `content_change` to get the `sent`, `pending` and `failed` values.